k8s-daemonset/helloworld/world.py

```diff
@@ -4,10 +4,11 @@
 app = Flask(__name__)
 
 pod_ip = os.getenv("POD_IP")
+target_world = os.getenv("TARGET_WORLD", "world")
```

Comment:
Yeah, this is slightly more complicated because I didn't want the blue-green deploy example to have to publish new docker images.  So at deploy time, we copy the contents of `world.txt` into the `TARGET_WORLD` environment variable, which is then read by this code.  That allows us to update the app without needing to update the docker image.  Here's where the environment variable is set:

https://gist.github.com/klingerf/14a78b3408eab0327b0de483dc174fbb#file-blue-green-groovy-L13